### ISSUE TITLE
Unit tests failing

### ISSUE THREAD

**Issue by jmeyers314** [rmjarvis/Piff#90]
`test_chipnum()` is failing for me with

```
E           galsim.errors.GalSimConfigError: No input image_file_name available for type = Current
E           Error generating Current value with key = image_file_name
```

This is master piff and master GalSim.

**Comment by rmjarvis** [comment]
Thanks for checking.  I fixed this in (GalSim's) releases/2.2 branch.  We can make v2.2.1 with this fix.  But let's fix your problems with clang first.

**Comment by rmjarvis** [comment]
I believe this is fixed now with the release of GalSim 2.2.1.
